src/Controls/tests/UITests/Tests/Issues/Issue19806.cs

```diff
@@ -0,0 +1,26 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues
+{
+	public class Issue19806 : _IssuesUITest
+	{
+		public Issue19806(TestDevice device)
+			: base(device)
+		{ }
+
+		public override string Issue => "Button doesn't respect LineBreakMode";
+
+		[Test]
+		public void TextInButtonShouldBeTruncated()
+		{
+			this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.Android | TestDevice.Windows | TestDevice.Mac });
```

Comment:
I see you are ignoring all platforms except mac catalyst. This looks like a core change, so I would suggest enabling it for all platforms.

Reply:
Done :)